tests/cli/test_schema_generation.py

```diff
@@ -97,11 +150,34 @@ def test_gen_pipeline_schema_without_custom_module(self, snapshot: SnapshotTest)
             catch_exceptions=False,
         )
 
+        assert result.exit_code == 0
+        assert len(result.stdout) > 0
```

Comment:
do you think a snapshot for the internal components would make sense?

Reply:
Not really, that's the main reason for adding the external components flag. I don't want the snapshot to update each time a component changes slightly.

Besides, the internal components schema is generated in the pre-commit hooks, so in a way, it replaces the snapshot test imo.

Reply:
makes sense